packages/server/graphql/mutations/helpers/notifications/SlackNotifier.ts

```diff
@@ -297,5 +309,85 @@ export const SlackNotifier: Notifier = {
 
   async integrationUpdated() {
     // Slack sends a system message on its own
+  },
+
+  async shareTopic(
+    dataLoader: DataLoaderWorker,
+    userId: string,
+    teamId: string,
+    meetingId: string,
+    reflectionGroupId: string,
+    stageIndex: number,
+    channelId: string
+  ) {
+    const r = await getRethink()
+    const [user, team, meeting, reflectionGroup, reflections, slackAuth] = await Promise.all([
+      dataLoader.get('users').load(userId),
```

Comment:
+1 I don't think we need to check if the `user` (the viewer) exists. We're already checking if the viewer is a teamMember in the `shareTopic` mutation